### PR TITLE
Feature/footer

### LINKED ISSUE #4: Footer
- [ ] [Standard](https://invis.io/N4FDSKWX8ZB#/279103827_Footer-Standard)
- [ ] [Light](https://invis.io/N4FDSKWX8ZB#/279353745_Footer-Light)

### PR DESCRIPTION
**Pull request by knuch**
fix #4 .

This branch was dependent on #21 So I merged the actual clean state of `feature/social` in this branch.

To allow you seeing the actual diff in github interface, I targeted that PR to `feature/social` branch.

**IMPORTANT** 
After review, wait for `feature/social` to be merged before merging into dev

### CODE REVIEW

File: assets/components/organisms/footer/footer-buttons.twig [hunk 1]
```diff
@@ -0,0 +1,4 @@
+<p class="footer-title footer-title-no-underline">Portail interne de l’EPFL</p>
+<a href="#" class="btn btn-secondary">Enseignants et chercheurs</a>
```
Comment (Yago) [inline assets/components/organisms/footer/footer-buttons.twig]: Those buttons should be `.btn-sm`

File: assets/components/organisms/footer/footer.scss [hunk 2]
```diff
@@ -0,0 +1,252 @@
...
+    transition: 0.2s;
+
+    li:last-child {
+      margin-bottom: 1rem;
+    }
+  }
+
+  .footer {
+    .active {
+      .footer-title {
```
Comment (Yago) [inline assets/components/organisms/footer/footer.scss]: Group those 3 into a single and less deep `.footer .active .footer-title` or better, something like (to keep things grouped) : 

```scss
.footer-title {
  // [...]
  .footer .acctive &:before { /*...*/ }
  .footer .acctive &:after { /*...*/ }
}

.footer-links {
  // [...]
  .footer .acctive & { /*...*/ }
}
```

File: assets/components/organisms/footer/footer.scss [hunk 3]
```diff
@@ -0,0 +1,252 @@
...
+        &:before {
+          transform: scaleY(0);
```
Comment (Yago) [inline assets/components/organisms/footer/footer.scss]: nice effect 👍

File: assets/config/helpers.scss [hunk 4]
```diff
@@ -66,3 +66,10 @@
     background: $value !important;
   }
 }
+
+.offscreen {
```
Comment (Yago) [inline assets/config/helpers.scss]: For what are you using it for ?

**Comment by Yago** [comment]
Alignement can also be improved to be more in the word center

styleguide_et_epfl_dynamic_styleguide_preview_mode_-_invision

File: assets/config/helpers.scss (hunk 4 above)
Comment (knuch) [reply assets/config/helpers.scss]: basolutely nothig, it was a test for a nice mobile animation for different heights. removed

**Comment by knuch** [comment]
@Yago ready to go

**Comment by Yago** [comment]
gif

**Pull request merged**